frontend/catalyst/api_extensions/error_mitigation.py

```diff
@@ -182,6 +183,8 @@ def __init__(
         self.extrapolate = extrapolate
         self.folding = folding
 
```

Comment:
Should it be renamed ZNECallable?

Reply:
Oh I thought I did 🤔

Reply:
Must have been a merge error, since its present in 10f371015e6b8a54668d2b126effbf0496055fb9

Reply:
fixed